cmd/juju/model/export_test.go

```diff
@@ -71,9 +71,9 @@ func NewDumpDBCommandForTest(api DumpDBAPI, store jujuclient.ClientStore) cmd.Co
 }
 
 // NewExportBundleCommandForTest returns a ExportBundleCommand with the api provided as specified.
-func NewExportBundleCommandForTest(bundleAPI ExportBundleAPI, cfgAPI ConfigAPI, store jujuclient.ClientStore) cmd.Command {
-	cmd := &exportBundleCommand{newAPIFunc: func() (ExportBundleAPI, ConfigAPI, error) {
-		return bundleAPI, cfgAPI, nil
```

Comment:
Do I understand correctly that we don't need config api because that was only used to find the SLA for the model to be able to configure its budget? If so, its very nice to see this removed.

Reply:
That is correct, yes